src/gaen/nativeModule.ts

```diff
@@ -64,11 +64,12 @@ export const requestAuthorization = async (): Promise<string> => {
   return permissionsModule.requestExposureNotificationAuthorization()
 }
 
-export const getCurrentENPermissionsStatus = async (): Promise<
-  ENPermissionStatus
-> => {
+export const getCurrentENPermissionsStatus = async (
+  cb: (status: ENPermissionStatus) => void,
```

Comment:
we removed all the callbacks from the native modules, is this ok?

Reply:
good question - yes, this callback happens in the JS layer

Reply:
ok, great! thanks for the explanation :)